snipsync.config.yaml

```diff
@@ -25,6 +25,8 @@ origins:
     repo: samples-java
   - owner: temporalio
     repo: temporal-ecommerce
+  - owner: vkarpov15
+    repo: samples-node
```

Comment:
for temporal long term cleanliness we should only rely on samples repos we own here. TODO - move this to the official node repo

Reply:
Did we end up adding more samples? I couldn't find them here.
If we did and want to keep this origin I'd at least open an issue so it's tracked.

Reply:
@vkarpov15 please comment here